.github/workflows/velox_weekly.yml

```diff
@@ -99,3 +100,102 @@ jobs:
           sudo apt-get install -y openjdk-8-jdk
           export JAVA_HOME=/usr/lib/jvm/java-8-openjdk-amd64
           cd $GITHUB_WORKSPACE/ && ./dev/package.sh
+
+  build-on-openeuler:
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ "openeuler/openeuler:24.03" ]
+    if: ${{ startsWith(github.repository, 'apache/') }}
+    runs-on: ubuntu-24.04-arm
+    container: ${{ matrix.os }}
+    steps:
+      - uses: actions/checkout@v4
+      - name: build
+        run: |
+          dnf update -y && dnf install -y java-1.8.0-openjdk-devel sudo patch wget git perl
+          $SETUP install_maven
+          export JAVA_HOME=/usr/lib/jvm/java-1.8.0-openjdk
+          export MAVEN_HOME=/usr/lib/maven
+          export PATH=${PATH}:${MAVEN_HOME}/bin:${JAVA_HOME}/bin
+          cd $GITHUB_WORKSPACE/ && ./dev/package.sh
+
+  build-on-openeuler-vcpkg:
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ "openeuler/openeuler:24.03" ]
+    if: ${{ startsWith(github.repository, 'apache/') }}
+    runs-on: ubuntu-24.04-arm
+    container: ${{ matrix.os }}
+    steps:
+      - uses: actions/checkout@v4
+      - name: build
+        run: |
+          dnf update -y && dnf install -y java-1.8.0-openjdk-devel sudo patch wget git perl
+          $SETUP install_maven
+          export JAVA_HOME=/usr/lib/jvm/java-1.8.0-openjdk
+          export MAVEN_HOME=/usr/lib/maven
+          export PATH=${PATH}:${MAVEN_HOME}/bin:${JAVA_HOME}/bin
+          export VCPKG_PATH=/var/cache/vcpkg
+          export VCPKG_BINARY_SOURCES="clear;files,${VCPKG_PATH},readwrite"
+          cd $GITHUB_WORKSPACE/ 
+          ./dev/vcpkg/setup-build-depends.sh
+          mkdir -p ${VCPKG_PATH}
+          sed -i 's/--build_arrow=OFF/--build_arrow=ON/' dev/package-vcpkg.sh && ./dev/package-vcpkg.sh
+          mkdir -p $GITHUB_WORKSPACE/.m2/repository/org/apache/arrow/
+          cp -r /root/.m2/repository/org/apache/arrow/* $GITHUB_WORKSPACE/.m2/repository/org/apache/arrow/
+      - uses: actions/upload-artifact@v4
+        with:
+          name: velox-native-lib-openeuler-24-${{github.sha}}
+          path: ./cpp/build/releases/
+          if-no-files-found: error
+      - uses: actions/upload-artifact@v4
+        with:
+          name: arrow-jars-openeuler-24-${{github.sha}}
+          path: .m2/repository/org/apache/arrow/
+          if-no-files-found: error
+
+  tpc-test-openeuler:
```

Comment:
I think it may be better to simply start a docker to run this tpc-h test in the above `build-on-openeuler-vcpkg` job after the build. Then, no need to upload artifacts, but only need to mount some paths to the docker, e.g., mount .m2 to include arrow jars, and mount $GITHUB_WORKSPACE to include native libs at /cpp/build/releases/ and gluten source code.

By doing so, we can reduce the complexity.

Reply:
Why use mount? can't we just copy `Build gluten-it` and `Run TPC-H` steps and place them after the `build` step?

Reply:
@kevinw66, I meant let's start a new docker to run the test, `docker run xxx bash -c "xxx"`, which can expose any libs lacking issue in deployment environment. “mount" is just for convenience to use some binaries built by previous step.